packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1547,13 +1547,16 @@ export default class Meeting extends StatelessWebexPlugin {
 
   /**
    * Removes the remote stream on the class instance and triggers an event
-   * to developers
+   * to developers for both remote stream and remote share
    * @returns {undefined}
    * @public
    * @memberof Meeting
    */
   unsetRemoteStream() {
-    this.mediaProperties.unsetRemoteStream();
+    // Unset both remote stream and remote share
```

Comment:
Thanks for adding the comment, but seems like it's a note for the PR review? Maybe this comment shouldn't be part of the code base. With the JSDoc update you made I think the change should be covered

Reply:
Appears to be convention in the code file to add comments in parts that later on down the road might raise an eyebrow. If you scroll above there's TODOs and other various comments in the code. Still if you prefer I can remove it.

Reply:
I think we should remove it, unless like you mentioned, there is a `TODO` item.

Reply:
Yes there are other TODOs and comments that already existed, I was just attempting to follow convention. Okay I removed it.